src/screens/MainScreen/index.js

```diff
@@ -1,17 +1,25 @@
 import React, { Component } from 'react'
-import { Text, View } from 'react-native'
+import { Button, View } from 'react-native'
+import { connect } from 'react-redux'
+import { logout } from '../../actions'
 import styles from './styles'
 
 class MainScreen extends Component {
 
+  logoutRequest = async () => {
+    await this.props.logout()
+    this.props.navigation.navigate('SignIn')
```

Comment:
```suggestion
    this.props.navigation.navigate(LOGIN_SCREEN)
```
You have a constant defined in screens.js